README.md

```diff
@@ -36,6 +36,31 @@ projects. This tool does not only detect them, but also fixes them for you.
 
 ## Documentation
 
+### Run with docker
+
+You can take a ready built docker image to run ``php-cs-fixer``.
```

Comment:
how will custom tokens act with docker image?
is it possible? if not out of the box, please describe how (or claim, in docs, lack of support)

Reply:
If you mean custom rules, then these will work normally because you need to mount whole project as a volume inside the Docker container, to run Fixer against the code. So effectively you mount it with your code and vendors. It's basically out of the scope of the PR because it's about using Docker properly and we're not here to teach it 😉.

Reply:
awesome. I was afraid some magic may be needed or not possible.
if business as usual, all great.